examples/reason-act.sh

```diff
@@ -1,15 +1,18 @@
+#!/usr/env/bin bash
 
-#!/bin/bash
+while getopts "m:" opt; do
+  case $opt in
+    m) MODEL="-m $OPTARG";;
+    \?) echo "Invalid option -$OPTARG" >&2; exit 1;;
+  esac
+done
 
-cd `dirname $0`
-cd ..
+shift $((OPTIND-1))
 
-# get -m model parameter otherwise defer to default
-if [ "$1" == "-m" ]; then
-  MODEL="-m $2 "
-fi
+cd "$(dirname "$0")"
+cd ..
 
-./main $MODEL --color \
+./bin/main $MODEL --color \
```

Comment:
The example scripts are meant to be run from the source root, so this has to remain `./main`